File: Assignment1/weightedSum.cpp
#include <iostream>
using namespace std;

int main() {
    int n, sum=0, j=1;
    cout << "Enter how many numbers to enter: ";
    cin >> n;
    cout << "Enter " << n << " numbers: ";
    for (int i=0; i<n; i++) {
        int x;
        cin >> x;
        if (x%2==0) {
            sum += (j*x);
            j++;
        }
    }
    cout << sum << endl;
    return 0;
}

File: Assignment1/weightedSum.java
import java.util.*;

public class weightedSum
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter how many numbers to enter: ");
		int n = sc.nextInt();
		int sum = 0;
		int j = 1;
		System.out.print("Enter " + n + " numbers: ");
		for (int i=0; i<n; i++)
		{
			int x = sc.nextInt();
			if (x%2==0)
			{
				sum += (j*x);
				j++;
			}
		}
		System.out.println(sum);

		sc.close();
	}
}